_episodes/03-use-registry.md

```diff
@@ -57,57 +57,54 @@ That is, metadata helps describe the software in a standardized way, so it can b
 {: .callout}
 
 
-## What are the existing / commonly used standards / descriptions for software metadata (Fatma)
-
-**Definition**
-A standard can be defined as:
-- A structure agreed and adopted by a community
-- A pattern or model used or accepted as normal or average
-- A standard may be represented by an Ontology, or a Controlled Vocabulary, etc
+## Existing and commonly used standards for metadata
 
-For capturing good meta-data, it is better to use an existing standard
+A standard can be defined as "a structure agreed and adopted by a community" or as "a pattern or model used and accepted as normal or average". Standards may be represented by an Ontology, or a Controlled Vocabulary, etc.
 
-> ## Ontology definition 
+> ## Ontology definition
 >
-> “In computer science and information science, an ontology encompasses a representation, formal naming, and definition of the categories, properties, and relations between the concepts, data, and entities that substantiate one, many, or all domains.” - Wikipedia (https://en.wikipedia.org/wiki/Ontology_(information_science)
+> "In computer science and information science, an ontology encompasses a representation, formal naming, and definition of the categories, properties, and relations between the concepts, data, and entities that substantiate one, many, or all domains." - Wikipedia (https://en.wikipedia.org/wiki/Ontology_(information_science)
 >
 {: .callout}
 
+TODO: difference between control vocabulary and ontology.
```

Comment:
Please have a look at this definition and associated examples https://semwebtec.wordpress.com/2010/11/23/contolled-vocabulary-vs-ontology/